protected/humhub/components/ActiveRecord.php

```diff
@@ -216,4 +217,45 @@ public function createValidators()
 
         return $validators;
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function setAttributes($values, $safeOnly = true)
+    {
+        if (is_array($values)) {
+            $schema = static::getTableSchema();
+
+            foreach ($values as $name => $value) {
+                // Make sure integers are stored with correct data type
+                $column = $schema->getColumn($name);
+                if ($this->columnValueCanBeNormalized($column, $value)) {
```

Comment:
In some situations `$column` could also be `null` if it is a non-database field. Maybe we should check this.

Reply:
Yes, it can be `null` and I did this check inside the method `columnValueCanBeNormalized()` https://github.com/humhub/humhub/pull/6379/files#diff-b0143204392e87bf27a8e461e98be6aba41b271e27f0186c04afb80e6cade955R251.